src/components/TheHeader.vue

```diff
@@ -20,11 +23,15 @@ const showBanner = computed(() => {
     <RouterLink :to="{ name: 'HomePage' }" class="logo">
       <LogoImage />
     </RouterLink>
-    <template v-if="showActions">
+    <template v-if="!loginView && !userStore.isLoggedIn">
```

Comment:
For better readability I'd suggest to created computed value out of it with better naming. For example `showHeaderForLoggedUser`